gcloud/pubsub/__init__.py

```diff
@@ -0,0 +1 @@
+# package
```

Comment:
That would be meta-ironic, as the copyright comment itself would be the only content, and is not itself copyrightable. :)

I intend to have the normal docstring and convenience imports there, after the rest of the package settles out.